wordpress-mobile/WordPress-iOS
# Site Monitoring: Implement WebKit-based Metrics view

Reimplement Metrics view using `WebKitViewController`.

Out of all the `WKWebView` wrappers in the app, I found the `WebKitViewController` works best for our use case. There are also `WPWebViewController` and `BlazeViewController,` but the former missed `custom user agent`, and the latter is Blaze-specific, at least in the name.

To test:

- Uninstall the app or logout
- Log in and select an Atomic site
- Verify the Metrics are loading

**Note**: I noticed an (existing) performance [issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/22497) with web views – will investigate it later if time allows.

**Known Issues**: No dark mode or dynamic type support.

![Simulator Screenshot - iPad (10th generation) - 2024-01-29 at 13 19 57](https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/718088d5-0a13-49af-8a37-ef1ad3d38918)

## Regression Notes
1. Potential unintended areas of impact: Site Monitoring
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
